UKHomeOffice/cerberus-service
# COP-4093: Implement dummy RoRo Freight form

## Description

Implement a dummy (that doesn't send any data) RoRo Freight form.

## To Test
1. Open the `Issue a target form`
2. Check if the form has the same fields as the old RoRo Freight form (link is in the Jira ticket).

## Developer Checklist

\* Required
